src/ensembl/src/content/app/browser/browser-nav/BrowserNavBar.test.tsx

```diff
@@ -1,46 +1,21 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render } from 'enzyme';
 
-import { BrowserNavBar } from './BrowserNavBar';
-import { BrowserNavStates, ChrLocation } from '../browserState';
-
-import styles from './BrowserNavBar.scss';
+jest.mock('./BrowserNavBarControls', () => () => (
+  <div>BrowserNavBarControls</div>
+));
+jest.mock('./BrowserNavBarMain', () => () => <div>BrowserNavBarControls</div>);
```

Comment:
Is this supposed to be `<div> BrowserNavBarMain </div>`?

Reply:
✔️